Added Stellar spec (CAIP-28) (#44)
ChainAgnostic/CAIPs: CAIPs/caip-28.md

---
caip: 28
title: Blockchain Reference for Stellar Namespace
author: Gleb Pitsevich (@pitsevich)

> discussions-to: TBD

Reviewer: you can refer this exact PR as a discussions url

status: Draft
type: Standard
created: 2021-02-17
requires: 2
---

## Simple Summary

This document is about the details of the Stellar network namespace and reference for CAIP-2.

## Abstract
In CAIP-2 a general blockchain identification scheme is defined. This is the
implementation of CAIP-2 for the Stellar network.

## Motivation
See CAIP-2.

## Specification

> ### Filecoin Namespace

Reviewer: rename this to "Stellar Namespace"

The namespace "stellar" refers to the wider Stellar ecosystem.

#### Reference Definition

The reference relies on Stellar's current designation of addresses belonging to test or main networks by prefixing them
with `testnet` or `pubnet` correspondingly.

Reference could only be populated with `testnet` or `pubnet` symbols.

> 

Reviewer: Missing "Resolution Method" section which I'm assuming was also missing on the Filecoin spec 

Refer to [EIP155 namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-3.md) for an example using JSON-RPC and [Cosmos namespace](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-5.md) for an example using REST

Author: @pedrouid Thanks for the feedback, added this section.

## Rationale

Blockchains in the "stellar" namespace are [two Stellar public networks](https://developers.stellar.org/docs/glossary/network-passphrase/) - pubnet and testnet.

## Backwards Compatibility

Not applicable

## Test Cases

This is a list of manually composed examples

```
# Testnet
stellar:testnet

# Mainnet
stellar:pubnet
```

## Links

> - [Filecoin Specification](https://developers.stellar.org/docs)

Reviewer: rename to "Stellar Specification"

---
caip: 28
title: Blockchain Reference for Stellar Namespace
author: Gleb Pitsevich (@pitsevich)
discussions-to: https://github.com/ChainAgnostic/CAIPs/pull/44#pullrequestreview-594204708
status: Draft
type: Standard
created: 2021-02-17
requires: 2
---

## Simple Summary

> This document is about the details of the Stellar network namespace and reference for CAIP-2.

Reviewer 2: ```suggestion
This CAPI defines the Stellar network namespace and reference for CAIP-2.
```

Author: @ligi Made the change

Reviewer 2: And merged ;-)
